book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,67 @@
+the data collection process, the dataset composition and pre-processing, and legal and ethical considerations when using the data.
+These resources helps contextualise the resource and is an opportunity for dataset creators to share any concerns, suggestions,
+and other advice regarding responsible use of the dataset.
+The Datasheets for Datasets paper contains a list of example questions that data creators should ask themselves and document.
+
+## Data Management
+
+Once data collection is complete, challenges emerge around data storage, access, and making changes.
+Because state-of-the-art ML datasets are very large, they tend to be stored in a central location, often a cloud provider database, and made available for download under certain circumstances.
+In addition to creating a governance structure for the opening or gating access to certain users, there is the challenge of creating a governance structure to allow people to request to opt-out their data from the dataset.
+Because opt-out occurs after a dataset has already been created, it will not remove the data from original versions of the dataset, but can facilitate removal from future versions.
+
+### Governance Tool: [Am I In the Stack?](https://huggingface.co/spaces/bigcode/in-the-stack)
+This tool was developed by the [BigCode](https://www.bigcode-project.org/) team to help developers inspect The Stack dataset so they cansee whether any of their repositories have been included and might be used for training ML models. 
+If that is the case, developers can fill out a form to request to opt-out from The Stack and have their data removed for future iterations of the dataset.
+
+### Governance Tool: [Have I Been Trained?](https://haveibeentrained.com/)
+This tool was developed by [Spawning](https://spawning.ai/) to help artists see if their work is a part of common datasets used to train AI art models and to opt-in or 
+opt-out to future training. They maintain a "registry of non-consenting data" where individuals and companies can check to see if their data is a part of these datasets.
+
+## Data Processing
+
+In order to address some of the problematic aspects of the data, such as biased or harmful content or existence of personal information, some data processing should be done before the dataset is used for model training, released to a wider audience, or ideally, even before stored to a database.
+Addressing these data quality and privacy issues may be motivated by regulatory compliance like GDPR or a team’s norms and best practices.
+Because this work, particularly in identifying and removing harmful content, may be contextual or reliant on the expertise of specific communities, these tasks are often performed in collaboration with community experts.
+As an example, when identifying sensitive information for removal from a dataset, a team may identify clear targets regarding an individual;s data such as usernames, passwords, and emails.
```

Comment:
```suggestion
As an example, when identifying sensitive information for removal from a dataset, a team may identify clear targets regarding an individual's data such as usernames, passwords, and emails.
```

Reply:
Should it maybe be mentioned that many of these can be automated? i.e. validators can check for emails?

EDIT: I see this is covered in the PII section later :)